backend/src/index.js

```diff
@@ -26,10 +26,14 @@ function handleTerminate(server) {
   process.on('SIGINT', exitHandler.bind(null, { exit: true }));
   process.on('SIGUSR1', exitHandler.bind(null, { exit: true }));
   process.on('SIGUSR2', exitHandler.bind(null, { exit: true }));
-  process.on('uncaughtException', exitHandler.bind(null, { exit: true }));
+  process.on('uncaughtException', (error) => {
+    // eslint-disable-next-line no-console
+    console.log('Uncaught exception: ', error);
+    exitHandler({ exit: true });
+  });
 }
 
-function startServer({ port = process.env.PORT } = {}) {
+function startServer({ port = process.env.PORT || 5000 } = {}) {
```

Comment:
in the README you took out the port so it feels a little weird that the port is still configurable here